content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -11,10 +11,11 @@ in the `Pending` [phase](#pod-phase), moving through `Running` if at least one
 of its primary containers starts OK, and then through either the `Succeeded` or
 `Failed` phases depending on whether any container in the Pod terminated in failure.
 
-Whilst a Pod is running, the kubelet is able to restart containers to handle some
-kind of faults. Within a Pod, Kubernetes tracks different container
+Within a Pod, Kubernetes tracks different container
 [states](#container-states) and determines what action to take to make the Pod
-healthy again.
+healthy again. Whilst a Pod is running, the kubelet is able to restart containers to
+handle some kind of faults. Pods are not restarted, they are terminated and respawned
+(recreated).
```

Comment:
```suggestion
handle some kind of faults. The kubelet doesn't restart unhealthy _Pods_; it terminates and then recreates the Pods.
```

Reply:
The kubelet doesn't **recreate** failed Pods. The control plane sometimes does that though - specifically, when some higher level API such as a ReplicaSet decides that this should happen.

Reply:
+1